src/css/controls/imports/settings-menu.less

```diff
@@ -107,7 +107,10 @@
 
     &:focus {
         font-weight: bold;
-        outline: none;
+
+        &&:not(.jw-no-focus) {
```

Comment:
outputs to `.jw-settings-content-item:focus.jw-settings-content-item:focus:not(.jw-no-focus) {` - should this be
`.jw-settings-content-item:focus:not(.jw-no-focus)` instead ?